sonar-batch/src/main/java/org/sonar/batch/scan/report/Reporter.java

```diff
@@ -19,9 +19,10 @@
  */
 package org.sonar.batch.scan.report;
 
-import org.sonar.api.BatchComponent;
+import org.sonar.api.BatchSide;
 
-public interface Reporter extends BatchComponent {
+@BatchSide
+public interface Reporter {
 
   public void execute();
```

Comment:
![MINOR](https://raw.githubusercontent.com/henryju/image-hosting/master/minor.png)"public" is redundant in this context.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2333)